src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/entity/PlanEntity.kt

```diff
@@ -39,12 +47,19 @@ class PlanEntity(
   @Enumerated(EnumType.STRING)
   var agreementStatus: PlanStatus = PlanStatus.DRAFT,
 
+  @CreatedDate
   @Column(name = "creation_date")
   val creationDate: Instant = Instant.now(),
 
+  @LastModifiedDate
   @Column(name = "updated_date")
   var updatedDate: Instant = Instant.now(),
 
+  @LastModifiedBy
+  @ManyToOne
+  @JoinColumn(name = "updated_by_id")
```

Comment:
Question: Do we want countersigning to 'count' as a modification?

Reply:
No, I don't think so - the plan is in a locked state at that point